src/nvimage/index.ts

```diff
@@ -1902,6 +1977,51 @@ export class NVImage {
         } // for z
       } // for t (time)
     } // for d (direction, phase/real, etc)
+    /*
```

Comment:
@neurolabusc , do you want to keep all of these commented lines in the code?

Reply:
@hanayik I do think we should keep the commented lines for a bit. The idea here is that we could mimic the features of [tensor2metric](https://mrtrix.readthedocs.io/en/dev/reference/commands/tensor2metric.html) to derive FA, ADC, V1, etc. from the images created by [dwi2tensor](https://mrtrix.readthedocs.io/en/dev/reference/commands/dwi2tensor.html). The challenge is to find a compact, fast and stable function for this. I describe this in [issue 1005](https://github.com/niivue/niivue/issues/1005). My sense is we keep the commented code for a bit, providing all the boilerplate to support different tensor2metric functions.